### DISCUSSION TITLE
Remote sessions disconnected after every ~60 seconds

### DISCUSSION THREAD

**Discussion by Creogenesis** [immense/Remotely#385] [Q&A] [answered]
Server:
**vmWare esxi > Ubuntu Server 20 > nginx** 
proxy_pass from main server to local sever (remotely)

In the browser noticed that the page refreshes every ~60 seconds. Is that normal? Is it supposed to be like this?
**Also, when I connect to the client session breaks at the same interval.** 
Can you tell me how to solve the problem? What settings should I look at? 
What logs can direct me to solve the problem?

p.s. find nginx log error:

upstream timed out (110: Connection timed out) while reading response header from upstream

**Comment by Creogenesis** [comment, marked as answer]
The solution is found. When you apply proxying to another server, you need to add directives to the nginx (remotely) server configuration file.
In each block:
location /
location /_blazor
location /AgentHub
location /ViewerHub
location /CasterHub

**proxy_send_timeout 150;
proxy_read_timeout 150;**

**Comment by CFox243** [comment]
> The solution is found. When you apply proxying to another server, you need to add directives to the nginx (remotely) server configuration file. In each block: location / location /_blazor location /AgentHub location /ViewerHub location /CasterHub
> 
> **proxy_send_timeout 150; proxy_read_timeout 150;**


are you able to elaborate on this?
im using nginx proxy manager and some connections time out; others do not.
i have a feeling you are referencing this.
![firefox_NydbyXQQNx](https://github.com/immense/Remotely/assets/128409855/5cdcfdbe-6173-44ea-b78b-37b7055464f7)
